maat-court-data-api/src/main/java/gov/uk/courtdata/assessment/controller/PassportAssessmentController.java

```diff
@@ -0,0 +1,94 @@
+package gov.uk.courtdata.assessment.controller;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import gov.uk.courtdata.assessment.service.PassportAssessmentService;
+import gov.uk.courtdata.assessment.validator.PassportAssessmentValidationProcessor;
+import gov.uk.courtdata.dto.ErrorDTO;
+import gov.uk.courtdata.dto.PassportAssessmentDTO;
+import gov.uk.courtdata.model.assessment.CreatePassportAssessment;
+import gov.uk.courtdata.model.assessment.UpdatePassportAssessment;
+import gov.uk.courtdata.util.DateUtil;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.annotation.PostConstruct;
+import java.time.LocalDateTime;
+
+@RestController
+@RequestMapping("/passport-assessments")
+@Slf4j
+@RequiredArgsConstructor
+@Tag(name = "Passport Assessments", description = "Rest API for passport assessments")
+public class PassportAssessmentController {
+
+    private Gson gson;
+    private final GsonBuilder gsonBuilder;
+    private final PassportAssessmentService passportAssessmentService;
+    private final PassportAssessmentValidationProcessor passportAssessmentValidationProcessor;
+
+    @PostConstruct
+    private void initialize() {
+        gson = gsonBuilder
+                .setPrettyPrinting()
+                .serializeNulls()
+                .registerTypeAdapter(LocalDateTime.class, new DateUtil.LocalDateTimeAdapter())
+                .create();
+    }
+
+    @GetMapping(value = "/{passportAssessmentId}", produces = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(description = "Retrieve a passport assessment record")
+    @ApiResponses(value = {
```

Comment:
This can be removed, the @ApiResponse annotation is repeatable now. We probably need to go back and change this in the existing controller too.